src/ophys_etl/modules/video/schemas.py

```diff
@@ -16,8 +17,23 @@ class VideoBaseSchema(argschema.ArgSchema):
             required=False,
             allow_none=True,
             default=3,
-            description=("Radius of median filter kernel; "
-                         "if None, no median filter applied"))
+            description=("Size of spatial filter kernel; "
```

Comment:
Our default median filter actually doesn't apply downsampling.

https://github.com/AllenInstitute/ophys_etl_pipelines/blob/main/src/ophys_etl/modules/median_filtered_max_projection/utils.py#L12-L41

The size of the frame coming in is the size of the frame coming out

```
>>> import numpy as np
>>> import scipy.ndimage as scipy_ndimage
>>> f = np.random.random((25, 25))
>>> 
>>> f2 = scipy_ndimage.median_filter(f, size=3, mode='reflect')
>>> f.shape
(25, 25)
>>> f2.shape
(25, 25)
```
(because this isn't complicated enough...)